.github/.env

```diff
@@ -8,3 +8,4 @@ KEY_VAULT=kv-sswwebsite-9eb3
 APP_INSIGHT_PROD=appInsight-sswwebsite-9eb3
 APP_INSIGHT_DEV=appInsight-sswwebsite-dev-9eb3
 NEXT_PUBLIC_CHATBASE_BOT_ID=gnbZXuG_5tgI_R3yzKcOn
+NEXT_PUBLIC_JOTFORM_ID=233468468973070
```

Comment:
ideally, this should be updatable in tina

Reply:
Fixed